src/p_floor.c

```diff
@@ -254,7 +254,7 @@ void T_MoveFloor(floormove_t* floor)
   );
   
   if (!(leveltime&7))     // make the floormove sound
-    S_StartSound((mobj_t *)&floor->sector->soundorg, sfx_stnmov);
```

Comment:
This flow works if there is a final sound that the soundorg makes when the loop should stop, right? I can see that for instance woof's floors do always make a sound - when they stop naturally. Is there anything that can interrupt a moving plane without making a sound? (In which case, the looping sound would continue in perpetuity.)

Reply:
Here's an example of the type of edge case that concerns me 😅 
```c
          case 163:
            // Stop Moving floor
            // 163 S1  EV_DoPlat(perpetualRaise,0)
            EV_StopPlat(line);
            P_ChangeSwitchTexture(line,0);
            break;
```